experimental/regression_suite/tests/pregenerated/test_llama2.py

```diff
@@ -191,6 +204,62 @@ def test_step_a100_vulkan_stripped(llama2_7b_f16qi4_a100_vulkan_vmfb):
     )
 
 
+###############################################################################
+# Correctness
+###############################################################################
+
+
+llama2_7b_f16qi4_first_input_cpu = fetch_source_fixture(
+    "https://storage.googleapis.com/shark_tank/llama_regression/llama2-7b-i4-golden-outputs/cpu/first_vicuna_forward_input.npy",
+    group="llama2_7b_f16qi4_first_input_cpu",
+)
+
+llama2_7b_f16qi4_first_output_cpu = fetch_source_fixture(
+    "https://storage.googleapis.com/shark_tank/llama_regression/llama2-7b-i4-golden-outputs/cpu/first_vicuna_forward_output.npy",
+    group="llama2_7b_f16qi4_first_output_cpu",
+)
+
+llama2_7b_f16qi4_second_input_cpu = fetch_source_fixture(
+    "https://storage.googleapis.com/shark_tank/llama_regression/llama2-7b-i4-golden-outputs/cpu/second_vicuna_forward_input.npy",
+    group="llama2_7b_f16qi4_second_input_cpu",
+)
+
+llama2_7b_f16qi4_second_output_cpu = fetch_source_fixture(
+    "https://storage.googleapis.com/shark_tank/llama_regression/llama2-7b-i4-golden-outputs/cpu/second_vicuna_forward_output.npy",
+    group="llama2_7b_f16qi4_second_output_cpu",
+)
+
+
+@pytest.mark.postsubmit
+@pytest.mark.unstable_linalg
+@pytest.mark.plat_host_cpu
+def test_correctness_host_cpu(
```

Comment:
Can you find and link in the CI logs where this is now running? I searched a bit in https://github.com/openxla/iree/actions/runs/6437229726/job/17482170040?pr=15131 but I don't see this.

I would also find it useful (in the PR description) to know how much time this adds to the CI runs.

Reply:
This currently just adds the recipe for this. I tested these locally and they work. I want to run these recipes on the CI in the next patch I send for this.